learning_journal/README.md

```diff
@@ -12,26 +12,24 @@ Getting Started
 ## URL to Heroku deployment
 https://vic-learning-journal.herokuapp.com/
 
-## Coverage output for step2
----------- coverage: platform darwin, python 2.7.12-final-0 ----------
-Name                                       Stmts   Miss  Cover   Missing
-------------------------------------------------------------------------
-learning_journal/__init__.py                   8      0   100%
-learning_journal/routes.py                     5      0   100%
-learning_journal/views.py                     14      0   100%
-------------------------------------------------------------------------
-TOTAL                                         27      0   100%
-
-24 passed in 1.11 seconds
-
-
----------- coverage: platform darwin, python 3.5.2-final-0 -----------
-Name                                       Stmts   Miss  Cover   Missing
-------------------------------------------------------------------------
-learning_journal/__init__.py                   8      0   100%
-learning_journal/routes.py                     5      0   100%
-learning_journal/views.py                     14      0   100%
-------------------------------------------------------------------------
-TOTAL                                         27      0   100%
-
+## Coverage output for step2  
+---------- coverage: platform darwin, python 2.7.12-final-0 ----------  
+Name                            |       Stmts   |Miss  |Cover  |Missing  
+:------------------------------:|:-------------:|:----:|:-----:|:-----:  
+learning_journal/__init__.py    |           8   |   0  |100%   |  
+learning_journal/routes.py      |           5   |   0  |100%   |  
+learning_journal/views.py       |           14  |   0  |100%   |  
+                                |               |      |       |  
+TOTAL                           |           27  |   0  |100%   |  
+24 passed in 1.11 seconds  
+
+
+---------- coverage: platform darwin, python 3.5.2-final-0 -----------  
+Name                            |        Stmts  |Miss  |Cover   |Missing  
+-------------------------------:|:-------------:|:----:|:------:|:-----:  
+learning_journal/__init__.py    |            8  |    0 |  100%  |  
+learning_journal/routes.py      |            5  |    0 |  100%  |  
+learning_journal/views.py       |            14 |    0 |  100%  |  
+                                |               |      |        |  
+TOTAL                           |            27 |    0 |  100%  |  
```

Comment:
Thanks for including htis, but it's not really needed.  Nice bragging rights, though.